frontend/assets/stylesheets/style.scss

```diff
@@ -109,6 +120,12 @@
 @import 'state/responsive';
 
 /* ==========================================================================
+   Theme
+   ========================================================================== */
+
+@import 'theme/tones';
```

Comment:
Tones now live in `theme/`. I matches SMACSS convention, and feels a little nicer than loading in on base/ file later on. These are theme/colour overrides after all.